packages/autocannon-ui-frontend/src/components/App.jsx

```diff
@@ -33,32 +34,63 @@ const useStyles = makeStyles(theme => ({
   },
   resultsGridItem: {
     marginTop: theme.spacing(2)
+  },
+  buttonsContainer: {
+    gap: '1rem'
   }
 }))
 
 function App() {
   const classes = useStyles()
 
-  const [results, setResults] = useState([])
+  const [results, setResults] = React.useState([])
+  const [selectedResults, setSelectedResults] = React.useState([])
+  const [isOpenCompareDialog, setIsOpenCompareDialog] = React.useState(false)
 
-  function resultsReceivedHandler(resultSet) {
+  const handleOnReceivedResults = React.useCallback(resultSet => {
     setResults(results => [...results, resultSet])
-  }
+  }, [])
+
+  const handleOnSelectResultSet = React.useCallback((resultSet, isSelected) => {
+    if (isSelected) {
+      setSelectedResults(results => [...results, resultSet])
+    } else {
+      setSelectedResults(results =>
+        results.filter(
+          item => item.start !== resultSet.start || item.url !== resultSet.url
+        )
+      )
+    }
+  }, [])
+
+  const toggleIsOpenCompareDialog = React.useCallback(() => {
+    setIsOpenCompareDialog(isOpen => !isOpen)
+  }, [])
 
   return (
     <ThemeProvider theme={theme}>
       <Container component="main" maxWidth="md">
         <CssBaseline />
 
-        <RunOptionsForm onNewResults={resultsReceivedHandler} />
+        <RunOptionsForm onNewResults={handleOnReceivedResults} />
 
         {results.length > 0 && (
           <Grid
             container
+            className={classes.buttonsContainer}
             direction="row"
             justifyContent="flex-end"
             alignItems="center"
           >
+            <Button
+              variant="contained"
+              color="primary"
+              onClick={toggleIsOpenCompareDialog}
+              disabled={selectedResults.length !== 2}
```

Comment:
Is not magic. 
Why should I extract a const if used once?

Reply:
Because it could be useful for who will came after you whats that 2 stands for, just to give a meaning of that number.
An example: https://refactoring.guru/replace-magic-number-with-symbolic-constant